fixtures/run_as_non_root_psc_true_csc_false_v1.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
```

Comment:
That also means we will most likely want more than one test for this. Because we need to check how the behavior is with more than one container and what happens if some of the containers have it set, others don't, etc.